dask/highlevelgraph.py

```diff
@@ -1219,6 +1219,22 @@ def to_graphviz(
         attrs = data_attributes.get(k, {})
         attrs.setdefault("label", label(k, cache=cache))
         attrs.setdefault("shape", "box")
+        n_tasks = len(hg.layers[k])
+        # print(n_tasks)
+        if n_tasks <= 10:
```

Comment:
I would like to see fontsize as a continuous variable, rather than these discrete "buckets" for different ranges.

If it's a bit too tricky for you to do this with log scaling, a good first step might be to make an implementation with a linear relationship between ntasks and fontsize.

Reply:
Oh yes. I thought of the same. For now, I just wrote something very simple. 

Once our plan is final, I will make it clean.